### ISSUE TITLE
Error while building APK

### ISSUE THREAD

**Issue by SincerelyFaust** [yanncabral/open_settings_plus#13]
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':open_settings_plus:compileReleaseJavaWithJavac'.
> error: invalid source release: 17

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
Running Gradle task 'assembleProductRelease'...                     5,8s
Gradle task assembleProductRelease failed with exit code 1
```

I can build iOS without a single issue however I get this while building the APK.

**Comment by yanncabral** [comment]
Does it work with the 0.3.1 version?

**Comment by SincerelyFaust** [comment]
I tried both 0.3.2 and 0.3.1 but no luck.

**Comment by yanncabral** [comment]
Hello @SincerelyFaust ,

I'm pleased to inform you that the issue you reported has been resolved in the latest release (version 0.3.3) of open_settings_plus. Thanks for your patience and for bringing this to our attention.

A special thank you to @lcmacedo for their contribution to the fix—it was immensely helpful.

Feel free to update to the new version and let us know if everything works as expected!

Best regards,
Yann Cabral
